src/components/MenuItem.jsx

```diff
@@ -27,19 +27,19 @@ function MenuItem() {
           <ListItem>
             <Link
               component={RouterLink}
-              to="/patients"
+              to="/employees"
               sx={{ textDecoration: "none" }}
             >
-              Archived Patients
+              Employee Management
             </Link>
           </ListItem>
           <ListItem>
             <Link
               component={RouterLink}
-              to="/employees"
+              to="/patients"
               sx={{ textDecoration: "none" }}
             >
-              Employee Management
+              InPatient Management
```

Comment:
this functionality on nav links seems out of scope

Reply:
Please i don't understand

Reply:
Oh if you look at the user story critically it's asking for button functionality. There is a key phrase in the criteria that says "all three buttons" .. not links. 
... I'm just wondering why the MenuItem showed up in this PR. Please advice why it showed up in the PR